lib_device_control/module_build_info

```diff
@@ -1,7 +1,7 @@
 VERSION = 5.0.0
 
-DEPENDENT_MODULES = lib_xassert(>=4.0.0) \
-                    lib_logging(>=3.0.0)
+DEPENDENT_MODULES = lib_xassert(>=4.2.0) \
```

Comment:
pin versions here

Reply:
This is not possible, see the error below during the library checks:

```
[2024-10-10T15:02:16.159Z]     def test_dependencies(dir):
[2024-10-10T15:02:16.159Z]         (res, msg) = do_deps_check(dir)
[2024-10-10T15:02:16.159Z] >       assert res, msg
[2024-10-10T15:02:16.159Z] E       AssertionError: Dependency lib_xassert(4.2.0) does not specify version
[2024-10-10T15:02:16.159Z] E       Dependency lib_logging(3.2.0) does not specify version
[2024-10-10T15:02:16.159Z] E       
[2024-10-10T15:02:16.159Z] E       module_build_info should specify dependencies with minimum versions e.g.
[2024-10-10T15:02:16.159Z] E       
[2024-10-10T15:02:16.159Z] E       DEPENDENT_MODULES = lib_logging(>=1.0.0) lib_ethernet(>=3.0.0)
```